e2e/cli_test.go

```diff
@@ -494,6 +494,20 @@ func TestCreateNewBuiltinRuleFromTemplate(t *testing.T) {
 	}
 }
 
+func TestCreateNewBuiltinRuleDataYamlAndReadme(t *testing.T) {
+	t.Parallel()
+
+	stdout := bytes.Buffer{}
+	stderr := bytes.Buffer{}
+
+	tmpDir := t.TempDir()
+
+	err := regal(&stdout, &stderr)("new", "rule", "--type", "builtin", "--category", "naming", "--name", "foo-bar-baz", "--output", tmpDir)
+	if exp, act := 1, ExitStatus(err); exp != act {
```

Comment:
Not sure I understand this test :) Do we expect the command to exit with an exit code of 1?

Reply:
Good question and sorry about the confusion. 
The working directory for e2e test is ./e2e. 
The new.go expects the project top level folder as the current working directory, otherwise it will fail with error message "data.yaml file not found.  Please run this command from the top-level directory of the Regal repository"

I once tried to change the e2e test's working directory, but the entire test stack is based on ./e2e directory, I'm not sure if I can do the change.
I also tried to use the full path in new.go instead of the relative path, but it requires to pass along the executable path.

Reply:
Ah, I see. I think you can just go ahead and remove this test then.

Reply:
Sure, I'm doing it right now.